thesis/inc/technical/detailed_architecture/component.tex

```diff
@@ -0,0 +1,43 @@
+\subsection{Component}
+\label{subsec:detailed_component}
+The component is a simple base class that all
```

Comment:
Somewhat wierd sentence. Would rather write "The component is a simple base class that all components must inherit from". I feel everything i discarded in the new sentence was just "fluff" and did not add anything to the old sentence.